src/components/Sponsors.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const SponsorCard = ({ className, children, style = {} }) => (
+  <div
+    className={`${className}`}
+  >
+    {children}
+  </div>
+);
+
+export default SponsorCard;
```

Comment:
This component quite literally doesn't do anything. Just remove it?

Reply:
I'll look into it.

Reply:
I think there needs to be a ja object, even an empty one for the graphql variable to work.